docs/getting-started/README.md

```diff
@@ -14,6 +14,38 @@ Welcome! Let's get started building with blocks. Blocks are at the core of exten
 
 -   [Create a Block Theme](/docs/how-to-guides/themes/create-block-theme.md) - Learn how block themes use blocks to build templates and the theme.json to provide styles.
 
+## Ways to Stay Informed
+
+New features and changes are important to keep up to date on as the Gutenberg project continues. Each person will have their own unique needs in keeping up with a project of this scale so what follows is more of a catalogue of ways to keep up rather than a recommendation for how to do so. 
```

Comment:
```suggestion
New features and changes are important to keep up to date on as the Gutenberg project continues. Each person will have their own unique needs in keeping up with a project of this scale. What follows is more of a catalogue of ways to keep up rather than a recommendation for how to do so. 
```